src/lintlist/mod.rs

```diff
@@ -875,6 +875,13 @@ pub const ALL_LINTS: [Lint; 343] = [
         deprecation: None,
         module: "methods",
     },
+    Lint {
```

Comment:
Hey you need to run `./util/dev update_lints` from the project root to update the lint count. Needs to be 344 now.

Reply:
yep, working on it now. I've been having trouble while working on this with certain tests failing because the stdout does not match. It is because there are a few extra lines like this:

```
-error[E0465]: multiple rlib candidates for `regex` found
-   |
-   = note: candidate #1: /c/Users/bsher/Documents/code/rust-projects/rust-clippy/target/debug/deps/libregex-4597a03e6ae80e63.rlib
-   = note: candidate #2: /c/Users/bsher/Documents/code/rust-projects/rust-clippy/target/debug/deps/libregex-4e8ce8deb48c4f7c.rlib
```

It usually resolves itself after a while or with a `cargo clean`, but I haven't been able to figure out exactly what the cause/fix is, any ideas?

Reply:
Yeah I have the same issue as well. Usually I just run `./test/ui/update-all-references.sh` and then ignore the modified regex test cases by not committing them but maybe there are other ways to deal with it that I am not aware of.

Reply:
okay cool, just wanted to make sure I wasn't doing something wrong. Thanks!

Reply:
Running `cargo clean` and recompiling solves this issue. Or just running `rm target/debug/deps/libregex*` should also solve this.

Another thing to do here is to only update the test, that is touched by your PR. The command to do that is in the output of a failed test. I think it's right below the stderr diff. To find it, you can grep for `update-references` (without `all`).